### DISCUSSION TITLE
useField stopped working when switching to uniforms-mui

### DISCUSSION THREAD

**Discussion by chiester** [vazco/uniforms#1245] [Q&A] [answered]
I'm trying to switch from uniforms-material to uniforms-mui. I have a form that was working with uniforms-mui. Structure is like: 
```
<AutoForm>

<MyComponent />

</AutoForm>
```

within my component, I have a `useField` to get the value of the field, such as 

` const [{ value }] = useField("someField", {})`

However, now, after only changing `uniforms-material` to `uniforms-mui` , I get: 

```
Invariant Violation: useForm must be used within a form.

Two most common reasons for this error are:
1. Component calling this function doesn't have a parent Form component in the tree.
2. A duplicate uniforms dependency is installed in node_modules.

For more info check FAQ: https://uniforms.tools/docs/faq/#useform-must-be-used-within-a-form
```

**Comment by chiester** [comment, marked as answer]
Fixed. It was a version mismatch between uniforms-mui and uniforms and uniforms-bridge-json-schema.

**Reply by radekmie** [reply]
That's what the second reason in the error is about. We could make add a case for that too.
